src/js/login/containers/Main.jsx

```diff
@@ -111,12 +136,18 @@ const mapStateToProps = (state) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    updateLoggedInStatus: (update) => {
-      dispatch(updateLoggedInStatus(update));
+    isUserRegisteredForBeta: (service) => {
+      dispatch(isUserRegisteredForBeta(service));
     },
     toggleLoginModal: (update) => {
       dispatch(toggleLoginModal(update));
     },
+    toggleSearchHelpUserMenu: (menu, isOpen) => {
+      dispatch(toggleSearchHelpUserMenu(menu, isOpen));
+    },
+    updateLoggedInStatus: (update) => {
+      dispatch(updateLoggedInStatus(update));
+    },
     getUserData: () => {
       getUserData(dispatch);
```

Comment:
This is the reason this `mapDispatchToProps` is so verbose. `getUserData` is not currently an action, so it needs to receive `dispatch` as an argument, but it would be more appropriate to have it as an action under `user-profile`. Once that's done, this `mapDispatchToProps` can just be an object with direct mappings.